css/properties/scroll-margin.json

```
@@ -15,12 +15,26 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": "68"
-            },
-            "firefox_android": {
-              "version_added": "68"
-            },
+            "firefox": [
+              {
+                "version_added": "90"
+              },
+              {
+                "version_added": "68",
+                  "partial_implementation": true,
+                  "notes": "Before version 90, scroll margin is not calculated correctly on <code>element.focus()</code>, see <a href='https://bugzilla.mozilla.org/show_bug.cgi?id=1708303'>bug 1708303</a>."
```

A few suggestions to this note text:

- I've tried expanding the summary of the issue, to minimize the likelihood that a reader will need to read the full bug to understand what's going on here
- I've shortened the URL, to partly fix the CI failure

```suggestion
                  "notes": "Before version 90, the <code>scroll-margin</code> property can cause an element's visibility to be incorrectly calculated for <code>element.focus()</code>. See <a href='https://bugzil.la/1708303'>bug 1708303</a>."
```

That said, there's one or thing which is unresolved by my suggestion. Either we should have either:

- one support statement with `"version_added": "68"`, no `partial_implementation`, and the text "Before version 90", or
- two support statements without the text "Before version 90"

I'm inclined to go with one support statement because, if I understand correctly, this issue with `scroll-margin` only affects this one API call.

---

Thank you for the fix.

I would go with two support statements because the fixed version don't have an annotation like we have in the safari and safari_ios entries.
Also, it keeps the UI clean on applications like https://caniuse.com/?search=scroll-margin